extensions/industry/healthcare/fhir/datatypes/address.example.1.json

```diff
@@ -0,0 +1,13 @@
+{
+   "address":[
```

Comment:
The outer block "address" should not be there. The example will look like this:
`
{
   "use":"home",
   "line":[
      "Bos en Lommerplein 280"
   ],
   "city":"Amsterdam",
   "postalCode":"1055RW",
   "country":"NLD"
}
`

Same for all other examples below.

Reply:
@anandphatak ok! I will fix all the examples. thanks